cls/TestCoverage/Utils.cls

```diff
@@ -391,6 +429,122 @@ ClassMethod GetClassLineExecutableFlags(pClassName As %String, ByRef pDocumentTe
 	}
 }
 
+ClassMethod CodeArrayToList(ByRef pCodeArray, Output pDocumentText As %List)
+{
+	set pDocumentText = $lb()
```

Comment:
Should be:
```
set pDocumentText = ""
for i=1:1:$get(pCodeArray(0)) {
    set pDocumentText = pDocumentText _ $ListBuild(pCodeArray(i))
}
quit
```

Reply:
%List is a linked list so concatenation just works. I'll show you how it works sometime, remind me.